gluon-ssid-changer/files/lib/gluon/ssid-changer/ssid-changer.sh

```diff
@@ -1,23 +1,22 @@
 #!/bin/sh
 
 # maximum simplyfied, no more ttvn rating
-check=$(batctl gwl -H|grep -v "gateways in range"|wc -l)
```

Comment:
why is this not needed any more?

Reply:
it's a noop since -H suppresses the header and even with headers there woldn't be no such text

Reply:
it has - otherwise you get 1 as result (wc -l) - which is wrong
and the srcipt would never do hostapd update while thinking there is at least 1 server

Reply:
ye, it's already changed